lib/rubocop/cop/generator/configuration_injector.rb

```diff
@@ -49,6 +49,7 @@ def find_target_line
 
             break index if badge.to_s < line
```

Comment:
Sorry for my late reply.

```suggestion
             return index if badge.to_s < line
```

Currently it returns `configuration_entires.size -1` always. Can you fix this problem?